src/runtime/virtcontainers/utils/utils.go

```diff
@@ -321,6 +321,7 @@ func WaitLocalProcess(pid int, timeoutSecs uint, initialSignal syscall.Signal, l
 	if initialSignal != syscall.Signal(0) {
 		if err = syscall.Kill(pid, initialSignal); err != nil {
 			if err == syscall.ESRCH {
+				logger.WithField("pid", pid).Warnf("kill encounters ESRCH, process probably already finished")
```

Comment:
```suggestion
				logger.WithField("pid", pid).Warnf("kill encounters ESRCH, process already finished")
```

Reply:
@fengwang666, please, just change the warn message, removing the "probably" from there.

Reply:
done!